proc/proc.go

```diff
@@ -151,6 +149,9 @@ func (dbp *Process) LoadInformation(path string) error {
 }
 
 func (dbp *Process) FindFileLocation(fileName string, lineno int) (uint64, error) {
+	if runtime.GOOS == "windows" {
```

Comment:
I take it back - `filePath.ToSlash` does what's needed here and in the other two places you called out.  I'll use that.